R/multitables.R

```diff
@@ -26,6 +26,41 @@ setMethod("[[", c("MultitableCatalog", "numeric"), function (x, i, ...) {
     getEntity(x, i, Multitable, ...)
 })
 
+#' @rdname catalog-extract
+#' @export
+setMethod("[[<-", c("MultitableCatalog", "character", "missing", "formula"),
+          function (x, i, j, value) {
+              stopifnot(length(i) == 1)
```

Comment:
You should be able to do 

    ds <- loadDataset(datasetReference(x))

and then you can do `formulaToQuery` with `ds`. `loadDataset` should all be hitting the query cache so it won't be expensive.

If a `datasetReference` method for `Multitable` doesn't exist, add one. See the couple of patterns that do exist in the codebase (for when there is a dataset_url in the entity or when there isn't and you have to do a dirty URL munge)